Formula/openjdk.rb

```diff
@@ -147,14 +148,15 @@ def install
     bin.install_symlink Dir["#{libexec}/openjdk.jdk/Contents/Home/bin/*"]
     include.install_symlink Dir["#{libexec}/openjdk.jdk/Contents/Home/include/*.h"]
     include.install_symlink Dir["#{libexec}/openjdk.jdk/Contents/Home/include/darwin/*.h"]
-  end
 
-  def post_install
-    # Copy JavaNativeFoundation.framework from Xcode after install to avoid signature corruption
     if Hardware::CPU.arm?
-      cp_r "#{framework_path}/JavaNativeFoundation.framework",
-           "#{libexec}/openjdk.jdk/Contents/Home/lib/JavaNativeFoundation.framework",
-           remove_destination: true
+      dest = libexec/"openjdk.jdk/Contents/Home/lib/JavaNativeFoundation.framework"
+      # Copy JavaNativeFoundation.framework from Xcode
+      # https://gist.github.com/claui/ea4248aa64d6a1b06c6d6ed80bc2d2b8#gistcomment-3539574
+      cp_r "#{framework_path}/JavaNativeFoundation.framework", dest, remove_destination: true
+
+      # Replace Apple signature by ad-hoc one (otherwise relocation will break it)
+      system "codesign", "-f", "-s", "-", "#{dest}/Versions/A/JavaNativeFoundation"
```

Comment:
Can we use the `MachO` gem for this?

Reply:
I'd rather not, because it passes arguments I don't think we want in this case: `--preserve-metadata=entitlements,requirements,flags,runtime`